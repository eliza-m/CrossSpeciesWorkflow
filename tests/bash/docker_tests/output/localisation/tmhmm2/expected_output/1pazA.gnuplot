set arrow from 1,1.11 to 123,1.11 nohead lt 4 lw 10
set key below
set title "TMHMM posterior probabilities for 1pazA"
set yrange [0:1.2]
set size 2., 1.4
#set xlabel "position"
set ylabel "probability"
set xrange [1:123]
# Make the ps plot
set term postscript eps color solid "Helvetica" 30
set output "./TMHMM_6/1pazA.eps"
plot "./TMHMM_6/1pazA.plp" using 1:4 title "transmembrane" with impulses lt 1 lw 2, \
"" using 1:3 title "inside" with line lt 3 lw 2, \
"" using 1:5 title "outside" with line lt 4 lw 2
exit
