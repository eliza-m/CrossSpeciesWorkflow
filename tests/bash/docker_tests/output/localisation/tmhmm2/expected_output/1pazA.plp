# 1pazA
# AA	inside	membr	outside
1 E	0.27064	0.00000	0.72936
2 N	0.27064	0.00000	0.72936
3 I	0.27064	0.00000	0.72936
4 E	0.27064	0.00000	0.72936
5 V	0.27064	0.00000	0.72936
6 H	0.27064	0.00000	0.72936
7 M	0.27063	0.00001	0.72936
8 L	0.27063	0.00001	0.72936
9 N	0.27063	0.00001	0.72936
10 K	0.27063	0.00001	0.72936
11 G	0.27063	0.00001	0.72936
12 A	0.27063	0.00001	0.72936
13 E	0.27063	0.00001	0.72936
14 G	0.27063	0.00001	0.72936
15 A	0.27063	0.00002	0.72935
16 M	0.27063	0.00002	0.72935
17 V	0.27063	0.00002	0.72935
18 F	0.27063	0.00002	0.72935
19 E	0.27063	0.00002	0.72935
20 P	0.27063	0.00002	0.72935
21 A	0.27063	0.00002	0.72935
22 Y	0.27063	0.00002	0.72935
23 I	0.27063	0.00002	0.72935
24 K	0.27063	0.00001	0.72935
25 A	0.27063	0.00001	0.72935
26 N	0.27063	0.00001	0.72935
27 P	0.27063	0.00001	0.72935
28 G	0.27063	0.00001	0.72935
29 D	0.27063	0.00001	0.72935
30 T	0.27063	0.00001	0.72935
31 V	0.27063	0.00001	0.72935
32 T	0.27063	0.00001	0.72935
33 F	0.27063	0.00001	0.72935
34 I	0.27063	0.00001	0.72935
35 P	0.27064	0.00001	0.72935
36 V	0.27064	0.00001	0.72935
37 D	0.27064	0.00000	0.72935
38 K	0.27064	0.00000	0.72935
39 G	0.27064	0.00000	0.72935
40 H	0.27064	0.00000	0.72935
41 N	0.27064	0.00000	0.72935
42 V	0.27064	0.00000	0.72935
43 E	0.27064	0.00000	0.72935
44 S	0.27064	0.00000	0.72935
45 I	0.27064	0.00000	0.72935
46 K	0.27064	0.00000	0.72935
47 D	0.27064	0.00000	0.72935
48 M	0.27064	0.00000	0.72935
49 I	0.27064	0.00000	0.72935
50 P	0.27064	0.00000	0.72935
51 E	0.27064	0.00000	0.72935
52 G	0.27064	0.00000	0.72935
53 A	0.27064	0.00000	0.72935
54 E	0.27064	0.00000	0.72935
55 K	0.27064	0.00000	0.72935
56 F	0.27064	0.00000	0.72935
57 K	0.27064	0.00000	0.72935
58 S	0.27064	0.00000	0.72935
59 K	0.27064	0.00000	0.72935
60 I	0.27063	0.00002	0.72935
61 N	0.27061	0.00004	0.72935
62 E	0.27052	0.00013	0.72935
63 N	0.27005	0.00061	0.72934
64 Y	0.26923	0.00144	0.72932
65 V	0.26866	0.00203	0.72931
66 L	0.26851	0.00219	0.72929
67 T	0.26846	0.00227	0.72928
68 V	0.26827	0.00250	0.72923
69 T	0.26819	0.00262	0.72919
70 Q	0.26778	0.00325	0.72896
71 P	0.26355	0.00827	0.72818
72 G	0.26093	0.01113	0.72794
73 A	0.25357	0.01886	0.72757
74 Y	0.25071	0.02177	0.72752
75 L	0.25028	0.02221	0.72751
76 V	0.25022	0.02227	0.72751
77 K	0.25020	0.02230	0.72749
78 C	0.24986	0.02276	0.72738
79 T	0.24968	0.02313	0.7272
80 P	0.24947	0.02385	0.72668
81 H	0.24929	0.02450	0.72621
82 Y	0.24898	0.02618	0.72485
83 A	0.24888	0.02656	0.72456
84 M	0.24886	0.02634	0.7248
85 G	0.24884	0.02610	0.72506
86 M	0.24884	0.02569	0.72547
87 I	0.24883	0.02535	0.72582
88 A	0.24883	0.02502	0.72616
89 L	0.24884	0.02487	0.72629
90 I	0.24886	0.02470	0.72644
91 A	0.24893	0.02330	0.72777
92 V	0.24904	0.02139	0.72956
93 G	0.24959	0.01491	0.7355
94 D	0.25057	0.00703	0.74241
95 S	0.25061	0.00601	0.74339
96 P	0.25066	0.00479	0.74455
97 A	0.25069	0.00443	0.74488
98 N	0.25083	0.00379	0.74538
99 L	0.25092	0.00356	0.74552
100 D	0.25275	0.00143	0.74581
101 Q	0.25286	0.00128	0.74587
102 I	0.25288	0.00124	0.74589
103 V	0.25295	0.00111	0.74594
104 S	0.25309	0.00087	0.74604
105 A	0.25344	0.00037	0.74619
106 K	0.25372	0.00001	0.74627
107 K	0.25373	0.00000	0.74627
108 P	0.25373	0.00000	0.74627
109 K	0.25373	0.00000	0.74627
110 I	0.25373	0.00000	0.74627
111 V	0.25373	0.00000	0.74627
112 Q	0.25373	0.00000	0.74627
113 E	0.25373	0.00000	0.74627
114 R	0.25373	0.00000	0.74627
115 L	0.25373	0.00000	0.74627
116 E	0.25373	0.00000	0.74627
117 K	0.25373	0.00000	0.74627
118 V	0.25373	0.00000	0.74627
119 I	0.25373	0.00000	0.74627
120 A	0.25373	0.00000	0.74627
121 S	0.25373	0.00000	0.74627
122 A	0.25373	0.00000	0.74627
123 K	0.25373	0.00000	0.74627
