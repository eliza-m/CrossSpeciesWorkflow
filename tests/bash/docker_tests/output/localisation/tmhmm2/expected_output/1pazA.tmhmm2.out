# 1pazA Length: 123
# 1pazA Number of predicted TMHs:  0
# 1pazA Exp number of AAs in TMHs: 0.56487
# 1pazA Exp number, first 60 AAs:  0.00041
# 1pazA Total prob of N-in:        0.27064
1pazA	TMHMM2.0	outside	     1   123
